src/main/resources/log4j2.properties

```diff
@@ -7,6 +7,13 @@ appender.console.name = STDOUT
 appender.console.layout.type = JsonTemplateLayout
 appender.console.layout.eventTemplateUri = classpath:EcsLayout.json
 
-rootLogger.level = info
+appender.file.type = File
+appender.file.name = FILE
+appender.file.fileName = /poduser/logs/disaster-recovery.log
```

Comment:
Maybe `/poduser/logs/dr2-disaster-recovery.log`?

Reply:
I think it's ok as it is. This will be sitting in old DRI land and is supposed to be completely separate from the DR2 process. Also, I've named everything on the box like this and I don't want to change it :smile: